build-scripts/build_compiler.js

```diff
@@ -29,30 +29,70 @@
  * commit. This is for regular integration testing of the compiler with the various tests in this repo's packages.
  * In this case the compiler will be built as a SNAPSHOT.
  */
-'use strict';
+"use strict";
 
-const ncp = require('ncp');
-const fs = require('fs');
-const path = require('path');
-const runCommand = require('./run-command');
+const ncp = require("ncp");
+const fs = require("fs");
+const path = require("path");
+const runCommand = require("./run-command");
+
+const compilerTargetName = "compiler_unshaded_deploy.jar";
+const compilerJavaBinaryPath = `./compiler/bazel-bin/${compilerTargetName}`;
+
+async function main() {
+  console.log(process.platform, process.arch, compilerVersion);
+
+  const { exitCode } = await runCommand(
+    "bazel",
+    [
+      "build",
+      `//:${compilerTargetName}`,
+      `--define=COMPILER_VERSION=${compilerVersion}`,
```

Comment:
Bazel has a color arg to force color detection `--color=yes`. Does it improve the log output?

Reply:
Done. That's nicer.